First release with X-Server

vladpona opened the pull request:
- Added `openssl` package to the Dockerfile
- Added `mc` (Midnight Commander - ncurses based file manager for text mode and xterm usage)
- Added `epel` repository for extra packages installation (X-Server, XRDP, Chromium etc)
- Added XFCE4 as default desktop
- Added pack of helpers packages from `epel` (all what needs for X-Server normal functions)
- Added D-Bus subsystem mandatory environment variable: DBUS_SESSION_BUS_ADDRESS
- Added XFCE4-session as default clients windows manager
- Added supervisord services wrapper (as that impossible to use systemd inside of docker)
- Added `xrdp` and `xrdp-sesman` services to the system with running under `supervisord`
- Added `chromium` brovser installation
- Added `machine-id` generation for D-Bus subsystem functionality
- Added copy of default `xorg.conf` to common place position. And made some small modifications to it. Mostly to disable attempts of direct devices usage. As it impossible in docker because docker doesn't have access to DEVFS
- Added random password generation for RDP user. And storing it in ~/.rdp_credentials
- Added wrapper script for run D-Bus, XRDP and entry point to /bin/zsh
- Added base XFCE4 configuration: Default panel, additional desktop shortcuts to Cromium and X-Term
- Added RDP port (3389/tcp) exposition for remote usage possibility

Issue #34

davidgatti commented:
Please Update the README.md also with the command of what port to expose, right now I'm doing this: `docker run -p 3389:3389 -it -h docker 0x4447:x`

And regarding the password, I tried to type it twice, and carefully, so I'm not sure if the problem is me not knowing how to read or type, or there is something else :)

When I try to connect I get this:

```
xrdp-sesman[344]: pam_unix(xrdp-sesman:auth): authentication failure; logname= uid=0 euid=0 tty=xrdp-sesman ruser= rhost=  user=davidgatti
xrdp[347]: [INFO ] xrdp_wm_log_msg: login failed for display 0

xrdp[347]: [INFO ] login failed for display 0

xrdp[347]: [ERROR] SSL_read: I/O error

xrdp[347]: [ERROR] xrdp_iso_send: trans_write_copy_s failed

xrdp[347]: [ERROR] SSL_shutdown: I/O error

xrdp[347]: [ERROR] Sending [ITU T.125] DisconnectProviderUltimatum failed
```

![Screen Shot 2021-09-03 at 4 57 12 AM](https://user-images.githubusercontent.com/411114/131944026-6bb7e323-1434-41b8-8e9c-5b6b309f695b.png)

vladpona commented:
Updated.
Odd errors as I told before: there must be NOT `display 0`
To avoid possible interference with another RDP instance please try to run the docker as I updated in readme:

`docker run -p 15050:3389 -it -h docker 0x4447:latest`

And then connect to rdp://localhost/15050

If there still will be authentication problem then please use attached testing modification of Dockerfile. There always will be password `test` :)
[Dockerfile.txt](https://github.com/0x4447-setup/0x4447_docker/files/7273233/Dockerfile.txt)

vladpona commented:
Added note about RDP access details storage
